scipy/special/add_newdocs.py

```diff
@@ -2163,6 +2163,287 @@ def add_newdoc(name, doc):
            Graphs, and Mathematical Tables. New York: Dover, 1972.
     """)
 
+add_newdoc("elliprc",
+    r"""
+    elliprc(x, y)
+
+    Degenerate symmetric elliptic integral.
+
+    The function RC is defined as [1]_
+
+    .. math:: R_{\mathrm{C}}(x, y) = \frac{1}{2} \int_0^{+\infty} (t + x)^{-1/2} (t + y)^{-1} dt = R_{\mathrm{F}}(x, y, y)
```

Comment:
This line and lines 2224, 2273, 2321, 2339, 2367, and 2385 are making the linter complain that they are too long (longer than 79 characters). Please try splitting them into display math across multiple comment lines ([documented here](https://www.sphinx-doc.org/en/1.7/ext/math.html)), e.g.
```
    .. math::
       R_{\mathrm{C}}(x, y) =
         \frac{1}{2} \int_0^{+\infty} (t + x)^{-1/2} (t + y)^{-1} dt
         = R_{\mathrm{F}}(x, y, y)
```

Reply:
Thank you. The long lines have been broken into shorter ones.